### PR TITLE
Status code 415 is not documented (#203)

### LINKED ISSUE #203: producer/msg endpoint Status code 415 is not documented.
### Description
The status code for trying to call the producer/msg endpoint with the wrong content type is not documented in the list of statuscodes in the api-documentation.

```
{'timestamp': 'Aug 28, 2020 1:48:17 PM', 'path': '/producer/msg', 'status': 415, 'message': "Content type 'application/octet-stream' not supported", 'error': 'Unsupported Media Type'}
```

listed in the swagger-documentation:
```
201 | Created
400 | Invalid event content
401 | Unauthorized
403 | Forbidden 
404 | RabbitMq properties not found 
500 | Internal server error
503 | Service Unavailable
```

### PR DESCRIPTION
**Pull request by z-sztrom**
Adds documention of HTTP status code 415.

### Applicable Issues

Fixes #203 

### Description of the Change

List of returned HTTP statuses has been extended by status 415.

### Sign-off

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.

Signed-off-by: Roman Szturc roman.szturc.ext@ericsson.com

### CODE REVIEW

File: wiki/markdown/statusCodes.md
Comment (e-backmark-ericsson) [inline wiki/markdown/statusCodes.md:10]: Why are these status codes called "internal"?

File: wiki/markdown/statusCodes.md
Comment (e-backmark-ericsson) [inline wiki/markdown/statusCodes.md:15]: Status code 415 should be mentioned in this table as well

**Review by e-backmark-ericsson — CHANGES_REQUESTED** [review]
Is it only 415 that is missing in the current list, or is it possible that REMReM publish could also return other status codes?

**Comment by z-sztrom** [comment]
> Is it only 415 that is missing in the current list, or is it possible that REMReM publish could also return other status codes?

Yes, only 415 was missing. I checked in source code.

File: wiki/markdown/statusCodes.md
Comment (z-sztrom) [reply wiki/markdown/statusCodes.md:11]: Removed the "Internal"

File: wiki/markdown/statusCodes.md
Comment (z-sztrom) [reply wiki/markdown/statusCodes.md]: Added

**Comment by e-backmark-ericsson** [comment]
@z-sztrom why was this PR closed?

**Comment by z-sztrom** [comment]
I unintentionally closed the pull request. Is it possible to undo it?

**Comment by z-sztrom** [comment]
Reopened the PR

**Comment by e-backmark-ericsson** [comment]
I can't see that anyone approved this PR. It doesn't look very good that you merge your own PR. Anyway, I approve the changes made here, but I'm not actually a maintainer of REMReM so I shouldn't have the only say on this matter. I'd like to see an approval from someone in the actual maintainers team on this PR as well.
cc @eiffel-community/eiffel-remrem-maintainers

**Comment by z-sztrom** [comment]
I expected that it can be merged as the button isn't grayed, as in gerrit... My bad. I'll be more careful next time. Can I undo the merge?

**Comment by e-backmark-ericsson** [comment]
Yes, you were allowed to, as there was no branch protection rules set up for the master branch on this repo. So you didn't actually do wrong according to GitHub. But I believe it is good practice to always get an approval from some (other) maintainer before merging any PR. I therefore now added basic branch protection to all REMReM repos and informed about that in a mail to all REMReM maintainers.
No need to revert the merge here. I expect we get some approving comment from some other maintainer in this thread.

**Comment by z-sztrom** [comment]
Thanks for your explanation and sorry for the trouble.

**Pull request merged**
